recipe/meta.yaml

```
@@ -1,47 +1,50 @@
 {% set name = "authzed" %}
-{% set version = "1.20.0" %}
+{% set version = "1.24.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name|lower }}-split
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 7610e6b897d4a61765739183e68ca3de32cbc6dc0dc1df247444ae129e58627d
+  url: https://github.com/{{ name }}/{{ name }}-py/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 48c6debdcbce2bfe897e2957addae86fe8f3697141615ec51e3cc8edbd878a18
 
 build:
-  number: 1
-  skip: True  # [py<38]
+  number: 0
+  skip: true  # [py<38]
```

```suggestion
  skip: true  # [py<39]
```
https://github.com/authzed/authzed-py/blob/v1.24.0/pyproject.toml#L6